lightblue/src/test/java/org/esbtools/eventhandler/lightblue/LightblueDocumentEventRepositoryTest.java

```diff
@@ -189,15 +202,16 @@ public void shouldReturnNonOverlappingSetsEvenAmongMultipleThreads() throws Ligh
 
             Future<List<LightblueDocumentEvent>> futureThread1Events = executor.submit(() -> {
                 bothThreadsStarted.countDown();
-                return thread1Repository.retrievePriorityDocumentEventsUpTo(100);
+                return thread1Repository.retrievePriorityDocumentEventsUpTo(15);
             });
             Future<List<LightblueDocumentEvent>> futureThread2Events = executor.submit(() -> {
                 bothThreadsStarted.countDown();
-                return thread2Repository.retrievePriorityDocumentEventsUpTo(100);
+                return thread2Repository.retrievePriorityDocumentEventsUpTo(15);
```

Comment:
Tangentially related to core work of this PR; using lower numbers of events in this test makes it easier to understand when something goes wrong.